code/datums/traits/negative.dm

```diff
@@ -260,6 +260,8 @@
 	name = "Nearsighted"
 	desc = "You are nearsighted without prescription glasses, but spawn with a pair."
 	value = -1
+	var/obj/item/glasses
+	var/where
```

Comment:
Move these below the existing vars.
Its general a good idea to have new vars below existing ones.

Reply:
I mean, I did it like that because Family Heirloom places those there, but checking again most quirks that have the new vars place them together below, by wanting to keep consistency I actually decreased it which is kinda funny